src/test/java/com/microsoft/aad/msal4j/HttpHeaderTest.java

```diff
@@ -21,8 +23,12 @@ public void testHttpHeaderConstructor(){
                 .applicationVersion("app-version")
                 .build();
 
+        IAcquireTokenParameters parameters = UserNamePasswordParameters
+                .builder(Collections.singleton("scopes"), "username", "password".toCharArray())
+                .build();
+
         RequestContext requestContext = new RequestContext(
-                app, PublicApi.ACQUIRE_TOKEN_BY_AUTHORIZATION_CODE, null);
+                app, PublicApi.ACQUIRE_TOKEN_BY_AUTHORIZATION_CODE, parameters);
```

Comment:
what is the point of adding this, I see no assert where this is checked

Reply:
Due to the changes in `HttpHeaders`, unless parameters were passed in this would lead to null ref.